src/widget/weffectchainpresetbutton.cpp

```diff
@@ -61,14 +61,15 @@ void WEffectChainPresetButton::populateMenu() {
         const ParameterMap loadedParameters = pEffectSlot->getLoadedParameters();
         const ParameterMap hiddenParameters = pEffectSlot->getHiddenParameters();
 
+        const QString effectSlotNumPrefix(QString::number(effectSlotIndex + 1) + ": ");
         auto pManifest = pEffectSlot->getManifest();
         if (pManifest == nullptr) {
-            m_pMenu->addAction(tr("Empty Effect Slot %1").arg(effectSlotIndex + 1));
+            m_pMenu->addAction(effectSlotNumPrefix + kNoEffectString);
             continue;
         }
 
         auto pEffectMenu = make_parented<QMenu>(m_pMenu);
-        pEffectMenu->setTitle(pEffectSlot->getManifest()->displayName());
+        pEffectMenu->setTitle((effectSlotNumPrefix + pEffectSlot->getManifest()->displayName()));
```

Comment:
This entire thing could be written a bit more concisely if we choose the consistent formatting.

Reply:
I've moved fetching the parameter lists down and set the title in the initializer
Sqashed, rebased. Please check if that's what you had in mind.

Reply:
Alright now?